src/utils/isAmp.js

```diff
@@ -1,5 +0,0 @@
-// Returns whether the page is an amp page or not. 'process.env.AMP' should be replaced by
-// webpack on compile-time with 'true' or 'false'
-export default function isAmp() {
-  return process.env.AMP;
```

Comment:
I still think this isn't really helpful since env vars should work fine (and since it's an *environment* variable, you'd want amp to be the only thing served in one environment, and non-amp the only thing in another)